mobile-browser-based-version/src/Task Definition/mnist.js

```diff
@@ -0,0 +1,229 @@
+import * as tf from "@tensorflow/tfjs";
```

Comment:
I believe it would be best if we try to avoid spaces in file and folder names. For example, it would be better if we change 'Task Definition' to 'task_definition'. It can cause some problems between operating systems, especially on Windows